### PR TITLE
Edited Meet the Team Page

### PR DESCRIPTION
**Pull request by BAAHUBALI7781**

### CODE REVIEW

**Comment by RitikaShrestha-me** [comment]
@BAAHUBALI7781 That's mostly repetition of code that you have added.

> _No description provided._

File: src/views/Team.vue [hunk 1]
```diff
@@ -1,9 +1,31 @@
...
+                </div>
+                <div class=team-content>
+                    <h4 class=title>Surajit Kumar Roy</h4>
+                    <span class="post">Project Instructor</span>
+                </div>
+                <ul class="social">
+                    <li><a href="#" target="_blank" title="Facebook Page"><i class="fab fa-facebook"></i></a></li>
+                    <li><a href="#" target="_blank" title="Facebook Page"><i class="fab fa-instagram"></i></a></li>
+                    <li><a href="#" target="_blank" title="Facebook Page"><i class="fas fa-envelope"></i></a></li>
+                    <li><a href="#" target="_blank" title="Facebook Page"><i class="fab fa-linkedin"></i></a></li>
```
Comment (shivam-51) [inline src/views/Team.vue]: 4 "Facebook page"? forgot to edit, perhaps

File: src/views/Team.vue [hunk 2]
```diff
@@ -215,6 +269,10 @@
                 flex-direction: row;
                 flex-wrap: wrap;
             }
+            .container1{
+                flex-direction: row;
+                flex-wrap: wrap;
+            }
             .our_team{
```
Comment (shivam-51) [inline src/views/Team.vue:262]: why is this...

File: src/views/Team.vue [hunk 3]
```diff
@@ -227,6 +285,10 @@
                 flex-direction: column;
 
             }
+            .container1{
+                flex-direction: row;
+                flex-wrap: wrap;
+            }
```
Comment (shivam-51) [inline src/views/Team.vue]: ...and this same?

**Review by shivam-51 — CHANGES_REQUESTED** [review]

File: src/views/Team.vue (hunk 1 above)
Comment (BAAHUBALI7781) [reply src/views/Team.vue]: Yeah, I forgot to edit!

File: src/views/Team.vue (hunk 3 above)
Comment (BAAHUBALI7781) [reply src/views/Team.vue]: I have resolved the error! That was by mistake

**Comment by shivam-51** [comment]
You have copied a HTML snippet and edited that for every user, instead create JS objects and use the various names, links as object attributes. Display it using `v-for`.
This is what you too pointed out @RitikaShrestha-me , isn't it?

**Comment by RitikaShrestha-me** [comment]
I just wanted to say that he had some code that was just copy pasted and not required. The our_team:hover part had same code as normal our_team part. Which was not adding value but just extra code.

> You have copied a HTML snippet and edited that for every user, instead create JS objects and use the various names, links as object attributes. Display it using `v-for`.
> This is what you too pointed out @RitikaShrestha-me , isn't it?

**Comment by BAAHUBALI7781** [comment]
Yeah! @RitikaShrestha-me you are right, I just wanted to add the border radius style while hovering and I did it by copying the above code which could have been avoided. Thank you for pointing that out.

**Comment by RitikaShrestha-me** [comment]
That's the only thing I was telling :) 

> Yeah! @RitikaShrestha-me you are right, I just wanted to add the border radius style while hovering and I did it by copying the above code which could have been avoided. Thank you for pointing that out.

**Pull request merged**